### ISSUE TITLE
Could you please add a way to change the party leader?

### ISSUE THREAD

**Issue by Bastrabun** [fishyWET/party#1]
You already allow other party members be made officer, could you please add a command that makes the current party leader either a normal member or an officer and a person of choice the new partyleader?

For my reference 1755

**Comment by wrynvera** [comment]
Hi, glad to see this few years old mod is still receiving some interest.

The issue you stated is a flaw due to the way I built the system, the party leader name is the actual identifying ID for the party - so there's no easy way to set a new leader, the only option would be to disband the party and start a new one.

To note, I actually have been working on a more advanced version of this that resolves such issues, hopefully I can do a release for it in the next few months.

**Comment by Bastrabun** [comment]
Yes, we're using this on my server quite heavily. I disabled a few things like partychat, but it's a core mechanic and will be even more important when we add classes for players and then faction wars :)

If we can help in any way just say how we can do that. Especially testing, concepts or "usecases from the wild"

**Comment by wrynvera** [comment]
Great, good to hear it, that was definitely an intended purpose for the mod.
I do actually have a working version of the rewrite, but let me polish and ensure it's stable and ill let you know.

**Comment by Bastrabun** [comment]
Looks like it was implemented here: https://github.com/neinwhal/party/commit/0afca39449460bba74093fc2e728b0bb76ba3976

Maybe you want to PR it?

**Comment by wrynvera** [comment]
Currently don't have the time to develop this further.
If any pulls are submitted, I will accept them accordingly.
